lib/sedna/backend/tensorflow/__init__.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tensorflow as tf
+from sedna.backend.base import BackendBase
+from sedna.common.file_ops import FileOps
+
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
```

Comment:
should not hard code TF_CPP_MIN_LOG_LEVEL, user may want to customize it

Reply:
fix